### ISSUE TITLE
Czech translation

### ISSUE THREAD

**Issue by mar-svo** [backuppc/backuppc#520]
Hello maintainers, 
here is updated Czech translation: [cz.pm.zip](https://github.com/backuppc/backuppc/files/14757448/cz.pm.zip)

Best regards, Martin

**Comment by mar-svo** [comment]
.. update (corrected e-mail "subject" headers):
[cz.pm.zip](https://github.com/backuppc/backuppc/files/14757882/cz.pm.zip)

**Comment by GWHAYWOOD** [comment]
Thanks for the update, and sorry I don't speak this language -- but I can update the language file in BackupPC's Github repository.

If Microsoft's translator is right, the corrections include changing in several places a word which means something like "spilled" to a word which means something like "running".  The only difference is an accent on the first character 's'  in the word.

Does that sound approximately correct?  I only ask to make sure we're on the same page, so to speak.

**Comment by mar-svo** [comment]
.. yes it's OK.
Thank you.

**Comment by GWHAYWOOD** [comment]
Fixed in 52d8d77.
Thanks, closed.

**Comment by GWHAYWOOD** [comment]
Do we need to revert the translation of the 'Subject' headers?

In the other language files that I've checked they are in English.  I suspect that's required by the mail software.

**Comment by mar-svo** [comment]
Yes please, keep 'Subject' header in English - it's code-word of email headers.

**Comment by GWHAYWOOD** [comment]
Done, thanks.
